client/src/containers/HomePage.tsx

```diff
@@ -117,56 +118,56 @@ class HomePage extends Component<HomePageProps, State> {
             )}
           </div>
           <div className="HomePage__samples">
-            <h5 className="text-center">... or view some sample portfolios:</h5>
+            <h5 className="text-center"><Trans>... or view some sample portfolios:</Trans></h5>
             <div className="container">
               <div className="columns">
                 <div className="column col-4 col-sm-12">
                   <div className="HomePage__sample">
                     <h6>
                       <Link to={this.state.sampleURLs[0]} onClick={() => {window.gtag('event', 'example-portfolio-1-homepage');}} >
-                        Kushner Companies / Westminster Management
+                        <Trans>Kushner Companies / Westminster Management</Trans>
```

Comment:
@sraby should these company names be localized, or is it OK for them to just be in English?

Reply:
definitely ok for them to just be English! Shouldn't all proper names stay in English?

Reply:
Yeah, that's what I thought... I wasn't sure if it would be more appropriate to e.g. replace "management" with the localized translation of that word or something, but if people only ever see it written "a&e management" elsewhere then it makes sense to keep it the same for us too!